woocommerce/wc-admin
# Adding client-side filter to navigation root url

Fixes https://github.com/Automattic/wc-calypso-bridge/pull/659

Adding client-side filter for navigation `rootBackUrl` to support a fix for [this issue in the bridge](https://github.com/Automattic/wc-calypso-bridge/issues/640).

### Detailed test instructions:

-   Checkout branch
-   Add a snippet of client-side code like this:
```
addFilter( 'woocommerce_navigation_root_back_url', 'plugin-domain', () => 'https://www.gezorad.com');
```
- Navigate to any WooCommerce page with the new navigation, and then click the "WordPress Dashboard" button.
- Confirm that you're taken to the altered URL
